components/src/core/components/Input/Select/select-input.scss

```diff
@@ -229,4 +229,8 @@
       overflow: hidden;
     }
   }
-}
\ No newline at end of file
+}
+
+.double-line-item-selected {
```

Comment:
Discuss with Mike, why this is needed.

Reply:
removed. I'll fix the styles after discussing with Mike